docs/checkit.js

```diff
@@ -800,7 +900,15 @@ CheckIt.prototype.createCalendar = function () {
         calendar,
         initP,
         buildP;
-        
+    
+    // TODO
+    // if someone clicked on save, to edit their title and/or start date and/or end date
+    // then the same calendar state needs to be used as the currently displayed calendar
+    // except with the edited components changed.
+    // So validateform checks that each entry is filled in and filled in correctly,
+    // but this is only needed when creating a brand new calendar
+    // If we are already looking at a calendar then we need to isolate just the components
+    // that were edited. 
```

Comment:
What? lol
I mean, I'd need to have the same checkedDays... unless they picked a start date that comes after a bunch of checked days.. then I'd have to not have those anymore I guess.